build.gradle.kts

```diff
@@ -11,6 +11,8 @@ val springDocVersion by extra("1.6.10")
 
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-data-r2dbc")
+  runtimeOnly("org.springframework.boot:spring-boot-starter-jdbc")
```

Comment:
Flyway doesn't have support for r2dbc yet so have to include jdbc for this reason

Reply:
Might it be worth adding a comment - it will confuse others?!